.github/workflows/ci.yml

```
@@ -230,3 +230,153 @@ jobs:
           notify_when: 'failure'
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
+
+  publish-fuel-core-binary:
+    name: Release fuel-core ${{ matrix.job.target }}
```

Matrix doesn't exist here, so the name shows up literally as `Release fuel-core ${{ matrix.job.target }}`.
```suggestion
    name: Release fuel-core binaries
```